spec/unit/set_current_spec.rb

```diff
@@ -33,6 +33,15 @@
     expect(progress.current).to eq(0)
   end
 
+  it "doesn't allow nil" do
+    progress = TTY::ProgressBar.new("[:bar]", output: output, total: 10)
+    expect { progress.current = nil }.to raise_error(
```

Comment:
Super nitpicky. I usually have expectations that raise errors in multiline format with curly braces. This would be consistent with other tests.